packages/stream-model-handler/src/account-relations-utils.ts

```diff
@@ -0,0 +1,48 @@
+import { JsonReference } from 'json-ptr'
+import type { JSONSchema } from 'json-schema-typed/draft-2020-12'
+
+type FieldSchema = Exclude<JSONSchema, boolean>
+
+const SUPPORTED_FIELD_TYPES = ['boolean', 'integer', 'number', 'string']
+
+export function validateSetFields(fields: Array<string>, modelSchema: JSONSchema.Object): void {
```

Comment:
Can we add a quick comment on what the "fields" array represents and what this function is doing?